jujugui/static/gui/src/test/test_controller_api.js

```diff
@@ -2077,4 +2077,181 @@ describe('Controller API', function() {
     });
   });
 
+  describe('grantModelAccess', function() {
+    it('grants users access', function(done) {
+      // Perform the request.
+      const users = ['dalek', 'drwho@external'];
+      const access = 'read';
+      controllerAPI.grantModelAccess(users, 'default', access, err => {
```

Comment:
'default' here is kind of misleading as it should be a model uuid. Maybe use 'uuid-1234' or similar.